CHANGELOG.md

```diff
@@ -14,6 +14,35 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
```

Comment:
So what's the deal? It's not like every counter should have `_total` suffix. I don't remember such requirement from promtool. Renames always bring inconveniences, but I don't see any motivation for  them here.

Reply:
https://github.com/prometheus/client_golang/blob/618194de6ad3db637313666104533639011b470d/prometheus/testutil/promlint/promlint.go#L177

Reply:
Well, ok...